### PR TITLE
[RW-7608][risk=no] Behavior for URL access by non-CT users to CT workspaces/resources

### PR DESCRIPTION
**Pull request by NehaBroad**
Update the forbidden exception message when user tries to access a workspace that has a different access tier

Screen Shot 2021-12-08 at 2 07 32 PM

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java [hunk 1]
```diff
@@ -346,7 +346,10 @@ private void validateWorkspaceTierAccess(DbWorkspace dbWorkspace) {
     List<String> accessTiers = accessTierService.getAccessTierShortNamesForUser(userProvider.get());
 
     if (!accessTiers.contains(workspaceAccessTier)) {
-      throw new ForbiddenException("User does not have access to the workspace tier");
+      throw new ForbiddenException(
+          String.format(
+              "User with username %s does not have access to the workspace '%s' tier",
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java:351]: How about:
`User with username joel does not have access to the 'controlled' access tier required by workspace 'CT Test'`

Comment (NehaBroad) [reply api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java:351]: just the first name for user?

Comment (jmthibault79) [reply api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java:351]: full email for username is probably better 👍

Comment (NehaBroad) [reply api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java:351]: Screen Shot 2021-12-09 at 12 46 41 PM

**Pull request merged**